### PR TITLE
Add Homework from Module 1

### PR DESCRIPTION
**Pull request by WojciechSikora**

### CODE REVIEW

File: Homeworks/Module_1/Task_1.2/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const csv = require('csvtojson');
+const path = require('path');
```
Comment (ArturEpam) [inline Homeworks/Module_1/Task_1.2/index.js:2]: TASK 1.3Rewrite the above-mentionedprograms to use babel(https://babeljs.io/)and ES6modules.
So I guess you should use import instead https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules

File: Homeworks/Module_1/Task_1.1/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+process.stdin.setEncoding('utf8');
+
+process.stdin.on('data', function (stringFromUser){
+    let reversedString;
+    reversedString = stringFromUser.split('').reverse().join('');
```
Comment (ArturEpam) [inline Homeworks/Module_1/Task_1.1/index.js:5]: not sure what is the purpose of splitting this declaration into 2 lines. Could be simply: ``` const reversedString = ...```

Comment (WojciechSikora) [reply Homeworks/Module_1/Task_1.1/index.js:5]: you are right -> changed

File: Homeworks/Module_1/Task_1.2/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        console.log('Error occured: ' + error);
+    };
+};
+
+async function writeCsvContentToTxt (readPath, writePath) {
+    const logger = fs.createWriteStream(writePath, {
+        flags: 'a'
+    });
+    try {
+        const jsonObject = await convertCsvToJson(readPath)
```
Comment (ArturEpam) [inline Homeworks/Module_1/Task_1.2/index.js:35]: are you going to add pipeline instead?

File: Homeworks/Module_1/Task_1.2/index.js (hunk 1 above)
Comment (WojciechSikora) [reply Homeworks/Module_1/Task_1.2/index.js:2]: Added Task_1.3 folder with changed file

File: Homeworks/Module_1/Task_1.2/index.js (hunk 3 above)
Comment (WojciechSikora) [reply Homeworks/Module_1/Task_1.2/index.js:35]: I will try to rewrite this script using pipeline

Comment (ArturEpam) [reply Homeworks/Module_1/Task_1.2/index.js:35]: master-yoda-quotes

Comment (WojciechSikora) [reply Homeworks/Module_1/Task_1.2/index.js:35]: bob-burgers-ok-fine

Comment (ArturEpam) [reply Homeworks/Module_1/Task_1.2/index.js:35]: nah pipelines are cool :)

Comment (WojciechSikora) [reply Homeworks/Module_1/Task_1.2/index.js:35]: I have created new file "index_pipeline.js" in Task_1.2
please check

**Review by ArturEpam — APPROVED** [review]

**Review by ArturEpam — APPROVED** [review]

**Pull request merged**